pkg/kubelet/cadvisor/cadvisor_linux.go

```diff
@@ -32,12 +32,6 @@ import (
 	_ "github.com/google/cadvisor/container/docker/install"
 	_ "github.com/google/cadvisor/container/systemd/install"
 
-	// Register cloud info providers.
-	// TODO(#76660): Remove this once the cAdvisor endpoints are removed.
```

Comment:
This says they should be removed after the endpoints are removed.  Should we wait until that is the case?  I believe they were just disabled this release.

Reply:
we should ask @tallclair ( https://github.com/kubernetes/kubernetes/pull/76912 )

Reply:
i would like confirmation from others as well before removing.

Reply:
The motivation for waiting until the endpoints were removed was that this is a breaking change. By waiting for the cAdvisor endpoint removal (which we want to do anyway) I was deferring the work of making the breaking change to cAdvisor.

So, the first question is, what is the status of the removal of the cAdvisor endpoints? Does it have a date? If not, what's blocking it? I think cAdvisor is not yet the longpole for cloud provider removal, so if the planned cAdvisor endpoint removal date is before the final cloud provider removal date we could differ this.

If we definitely want to remove this before the cAdvisor endpoints are removed, we should probably treat this as a breaking change and take the usual precautions & process. I don't know of anyone using the cAdvisor endpoints, but I don't think we have a way of confirming that.